components/brave_shields/browser/brave_shields_web_contents_observer_android.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_shields/browser/brave_shields_web_contents_observer.h"
+
+#include <map>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "brave/browser/android/brave_shields_content_settings.h"
+#include "chrome/browser/android/tab_android.h"
+#include "chrome/browser/sessions/session_tab_helper.h"
+#include "content/public/browser/web_contents.h"
+
+using content::WebContents;
+
+namespace brave_shields {
+// static
+void BraveShieldsWebContentsObserver::DispatchBlockedEventForWebContents(
+    const std::string& block_type, const std::string& subresource,
+    WebContents* web_contents) {
+  int tabId = 0;
+  TabAndroid* tab = TabAndroid::FromWebContents(web_contents);
+  DCHECK(tab);
+  if (tab) {
+    tabId = tab->GetAndroidId();
+  }
+  chrome::android::BraveShieldsContentSettings::DispatchBlockedEvent(
+    tabId, block_type, subresource);
```

Comment:
lint nit: continuation of a line should be 4 spaces. There are a few others in here like this